src/workerd/io/worker-entrypoint.c++

```diff
@@ -753,10 +753,10 @@ kj::Promise<WorkerInterface::CustomEvent::Result> WorkerEntrypoint::customEvent(
 }
 
 #ifdef KJ_DEBUG
-void requestGc(const Worker& worker) {
+void requestGc(kj::Arc<Worker> worker) {
```

Comment:
passing Arc instead of & makes ownership more explicit and less error-prone in async situations

Reply:
Well, this method isn't async, it's entirely synchronous, so making the caller increment the refcount is redundant.

But it's only used in debug so I don't care that much.

Reply:
Hm.. given the usage here and how it is called, I agree with @kentonv that the additional addRef feels unnecessary here. Seems to me this could possible be a `const kj::Arc<Worker>&` ref instead and the capture below can remain `[&]`

Reply:
I think @mikea is right that the only call site has to addRef anyway, and is able to move-in that reference, so we're not adding any addrefs after all.